examples/generate-services-imperative/app.yaml

```diff
@@ -0,0 +1,74 @@
+apiVersion: container.cnrm.cloud.google.com/v1beta1
+kind: ContainerCluster
+metadata:
+  name: example
+  namespace: service-project-id # kpt-set: ${service-project-id}
```

Comment:
We could but since the example already project annotations/elements in it, it's better to mention the setters instead. Plus, IMO its better to focus on generate-services and adding `set-project-id` in there may create confusion and/or complexity.

Reply:
IIUC we don't need to show these setters here. The function looks at the `apiVersion` and `kind` and then determine what Service does it need.
If the setters are not related here, we should drop it. Doing that can reduce the chance that it confuse the users.